frontend/src/components/header/private-elements/private-elements.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from 'react';
+
+import ArrowDown from '@assets/images/header/arrow-drop-down.svg';
+import DefaultAvatar from '@assets/images/header/default-avatar.png';
+
+import { IRemindersProps, Reminders } from '../reminders/reminders';
+import styles from './private-elements.module.scss';
+
+interface IPrivateComponentProps extends IRemindersProps {
+  avatar: string | undefined | null;
+}
+
+export const PrivateElements: React.FC<IPrivateComponentProps> = ({
+  favorites,
+  notifications,
+  comparisons,
+  avatar,
+}) => {
+  const [isMenuOpen, setIsMenuOpen] = useState(false);
+
+  return (
+    <div className={styles.information}>
+      <Reminders
+        favorites={favorites}
+        comparisons={comparisons}
+        notifications={notifications}
+      />
+      <div
+        className={styles.avatarCover}
+        onClick={(): void => setIsMenuOpen(!isMenuOpen)}
+      >
```

Comment:
It's not semantically correct to apply any action listeners to the non-interactive elements. Such listeners need to be applied to the `button`s, links and other native clickable elements. Please consider using the MUI's [IconButton](https://mui.com/material-ui/react-button/#icon-button) & [Avatar](https://mui.com/material-ui/react-avatar/#main-content)